scripts/spherical_data_figure.py

```diff
@@ -0,0 +1,25 @@
+import yt
+import yt_idefix
```

Comment:
this line is not strictly needed anymore, I would suggest replacing it with a comment to point out that `yt-idefix` is required to run the script.